rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/OperatorMetricsEndpointSteps.java

```diff
@@ -0,0 +1,32 @@
+package io.syndesis.qe.rest.tests.steps;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.syndesis.qe.utils.HttpUtils;
+import io.syndesis.qe.utils.OpenShiftUtils;
+import io.syndesis.qe.utils.TestUtils;
+
+import java.io.IOException;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.fabric8.kubernetes.api.model.Endpoints;
+import io.fabric8.kubernetes.client.LocalPortForward;
+
+public class OperatorMetricsEndpointSteps {
+
+    @When("verify whether operator metrics endpoint is active")
```

Comment:
For verification steps, it is better to use @Then instead of @When
https://cucumber.io/docs/gherkin/reference/ 
`Then step should use an assertion to compare the actual outcome (what the system actually does) to the expected outcome (what the step says the system is supposed to do).`